deckhouse-controller/pkg/addon-operator/kube-config/backend/module_config.go

```diff
@@ -35,6 +36,7 @@ import (
 type ModuleConfigBackend struct {
 	mcKubeClient     *versioned.Clientset
 	deckhouseConfigC chan<- utils.Values
+	moduleEventC     chan events.ModuleEvent
```

Comment:
I think we have to set the direction of the channel

Reply:
As I remember, if I set it to a send channel, I can't use the channel for module manager for getting events :(